.github/workflows/build.yml

```diff
@@ -137,19 +140,38 @@ jobs:
       - name: Set env
         run: |
           cat tag.txt >> $GITHUB_ENV
+      - name: Create cosign public key
+        run: |
+          echo -n "${{ secrets.COSIGN_PUBLIC_KEY }}" | base64 --decode > cosign.pub
```

Comment:
Does the public key need to be a secret? In the next step you upload it as a build artifact, but I wonder if you could just commit it to the repo where people can get it from some stable URL, for example, https://raw.githubusercontent.com/buildpacks/lifecycle/main/cosign.pub

Reply:
Updated in 1eecae9